src/admin-views/base_admin_panel.php

```diff
@@ -0,0 +1,67 @@
+<div id="tribe_panel_base" class="ticket_panel panel_base" aria-hidden="false">
+	<div class="tribe_sectionheader ticket_list_container">
+		<div class="ticket_table_intro">
+			<span class="ticket_form_total_capacity">
+				Total Event Capacity:
+				<span id="ticket_form_total_capacity_value">
+					<?php
+					switch ( $total_tickets ) {
+						case -1:
+							?><i><?php esc_html_e( 'unlimited', 'event-tickets' ); ?></i><?php
```

Comment:
TBH, I waffled - it's only italicised when it's unlimited, so I used the `<i>` - I could as easily add a class?